ultimates/index.jp.md

```diff
@@ -8,15 +8,17 @@ permalink: /ultimates/jp/
 ---
 
 <div style="background-color: #002 ; padding: 10px; border: 1px solid;">
-日本語が母国語じゃないし、記事をすべて日本語で詳しくできないと思います。添削してくれたら感謝します！</div>
+日本語が母語ではないので記事をすべて日本語するのは難しいと思います。添削をしてくださると嬉しいです。</div>
+#I thought ult pfs are common in Mana Dc nowadays. It originally sounded like only ppl in Ele do ult pfs but i added some nuance to it to be precise.
 
 # 絶シリーズ
-
-日本DCの中でエレDCしか野良絶をしません。それで、野良のやり方はE言語側からのプレイヤーさんから決まりました。文化差があって、言葉の壁があっても日本コミュニティにももっと野良絶が浸透してほしい！
+現状エレメンタルDCほど野良絶が活発なコミュニティは日本DCの中ではありません。
```

Comment:
**note:** We should update this line, since Mana does actually PF Ultimates now (but with different strats).

Reply:
fixed.